fiftyone/core/collections.py

```diff
@@ -949,7 +961,7 @@ def head(self, num_samples=3):
         Returns:
             a list of :class:`fiftyone.core.sample.Sample` objects
         """
-        return [s for s in self[:num_samples]]
+        return [s for s in self._get_first(limit=num_samples)]
```

Comment:
I'm curious: do these changes optimize `first()` and `head()`? Or are you just standardizing on `_get_first()` usage?

For `first()`, we're replacing iterating one sample on `pipeline=[]` with `pipeline=[{$sort: {_id: 1}}, {$limit: 1}]`.

For `head()` we're going from `pipeline=[{$limit: n}]` to `pipeline=[{$sort: {_id: 1}}, {$limit: n}]`.

My intuition is that both should perform the same. Definitely would be weird if `{$sort: {_id: 1}}` made things faster. And I'd hope that `{$sort: {_id: 1}}` wouldn't make things any slower since `_id` is indexed.